org.opent2t.sample.hub.superpopular/com.insteon.hub/js/package.json

```diff
@@ -21,8 +21,10 @@
   "dependencies": {
     "request": "^2.74.0",
     "request-promise": "^4.1.1",
-    "opent2t": "^1.0.1",
-    "opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.0.0",
+    "opent2t": "^1.0.2",
+    "opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.2.0",
```

Comment:
>"opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.2.0", [](start = 4, length = 62)

package.json for opent2t-onboarding-org-opent2t-onboarding-insteon has the current version as 1.0.0, late published to NPM is 0.0.4, so this won't resolve in an npm install.  I think that the onboarder version of 1.0.0 is wrong (it should be 0.x.0)